kvmagent/kvmagent/plugins/imagestore.py

```
@@ -75,14 +75,14 @@ def stop_backup_jobs(self, vm):
             return shell.call(cmdstr).strip()
 
-    def stop_vm_backup_jobs(self, vm):
+    def stop_vm_backup_jobs(self, vm, force=False):
         with linux.ShowLibvirtErrorOnException(vm):
-            cmdstr = '%s stopbak -domain %s -batbak' % (self.ZSTORE_CLI_PATH, vm)
+            cmdstr = '%s stopbak -force=%s -domain %s -batbak' % (self.ZSTORE_CLI_PATH, force, vm)
             return shell.call(cmdstr).strip()
 
-    def stop_volume_backup_job(self, vm, drive):
+    def stop_volume_backup_job(self, vm, drive, force=False):
         with linux.ShowLibvirtErrorOnException(vm):
```

Ruff

`linux` may be undefined, or defined from star imports

(F405)